libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -74,4 +76,36 @@ export class StripeManager {
       customer.tax?.automatic_tax === 'not_collecting'
     );
   }
+
+  /**
+   * If customer taxId does not match incoming taxId, update customer record with incoming taxId.
+   *
+   * @param customerId
+   * @param taxId
+   * @returns
+   */
+  async addTaxIdToCustomer(customerId: string, taxId: string) {
+    // fetch customer by customerId
+    const customer = await this.client.fetchCustomer(customerId);
+
+    // check that customer exists
+    if (!customer) {
+      throw new Error('customer does not exist');
```

Comment:
This should follow our library error pattern. See the corresponding *.errors.ts in each library for an example here

Reply:
Updated.